react-components/src/apps/admin/AllStats/AppStatsComponents/NavBarTabs.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Imported in AllStats.js. This function imports the files for all the tabs that are present under TabPanels
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+import { AppBar, Tabs, Tab, Typography, Box } from "@material-ui/core";
+import JobsTab from "./JobsTab";
+import AppsTab from "./AppsTab";
+import UsersTab from "./UsersTab";
+import ids from "./AllStatsIDs";
+import { getMessage, withI18N, build } from "@cyverse-de/ui-lib";
+import myMessagesFile from "./messages.js";
+
+function TabPanel(props) {
+    const { children, value, baseId, index, ...other } = props;
+
+    return (
+        <Typography
+            component="div"
+            role="tabpanel"
+            hidden={value !== index}
+            id={`nav-tabpanel-${index}`}
+            aria-labelledby={`nav-tab-${index}`}
+            {...other}
+        >
+            <Box p={3}>{children}</Box>
+        </Typography>
+    );
+}
+
+TabPanel.propTypes = {
+    children: PropTypes.node,
+    index: PropTypes.any.isRequired,
+    value: PropTypes.any.isRequired,
+};
+
+function a11yProps(baseId, index) {
+    return {
+        id: build(baseId, index),
+        "aria-controls": `nav-tabpanel-${index}`,
```

Comment:
Minor bug here with [aria-controls](https://www.w3.org/WAI/PF/aria-1.1/states_and_properties#aria-controls).  Basically `aria-controls` needs to match up with the `id` of the element it controls, and in your case they're mismatched.  `aria-controls` will have `nav-tabpanel-jobsTab`, while the element ID in `TabPanel` will be `nav-tabpanel-0`.  In other words, in `a11yProps`, `index` is not actually an index, it's part of the static ID from your IDs file, while in `TabPanel` `index` is actually an index.

Reply:
Good catch. I was assuming this was the same as an `aria-label` and could be any descriptive text.

Reply:
@ash3rz I had resolved this because I had fixed this by changing the function to this 
`function a11yProps(baseId, index) {
    return {
        id: build(baseId, index),
        "aria-controls": build(baseId, index),
    };
}`

I'm pretty sure that is not how I am supposed to do it but using this `aria-controls` matches with the `id`.
Let me know what I can do to fix this

Reply:
Yes, you have `aria-controls` matching the id of the current element, rather than the element it controls, the `Typography` within `TabPanel`.  If you double check the sample code from material-ui's Tab demo, you'll see what needs to match between the different `id`, `aria-labelledby`, and `aria-controls` props.

Reply:
Okay, I've made some changes that match the pattern shown in material ui's tab demo.

Reply:
Still not quite 😄 In MUI's example, you have basically 2 elements with their own distinct static IDs - the tab and the tab panel. Based on those 2 distinct IDs, you establish the `aria-controls` and `aria-labelledby` relationship between those 2 elements.
Basically, you need 
```
...
<Typography
  id=a
  aria-labelledby=b
  ...
/>
...
function a11yProps(baseId, index) {
  return {
    id: b
    aria-controls: a
  }
}
```